### ISSUE TITLE
<li> elements must be contained in a <ul> or <ol>

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#99]
This is an issue with both the header nav elements and the breadcrumbs.

<img width="1098" alt="Screen Shot 2020-04-01 at 4 49 33 PM" src="https://user-images.githubusercontent.com/31868510/78185918-d55fba80-7439-11ea-95a7-97399ee1b81c.png">

Here is the relevant code:

<img width="486" alt="Screen Shot 2020-04-01 at 5 01 04 PM" src="https://user-images.githubusercontent.com/31868510/78186250-6cc50d80-743a-11ea-9004-dad2f99e6cb5.png">

Semantically, an `<li>` element must be the child of a `<ul>` or `<ol>` element. In the case of the header menu, because of the `role="menu"` on the parent `<ul>`, assistive technologies do not see a list but rather a _menu_ . Because of this, `role="menuitem"` is required on the children `<a>` elements, and `role="none"` should be added to the children `<li>` elements. You can see an example of proper implementation here:

https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

Similarly, the breadcrumb `<ol>` has a `role="navigation"`. This is semantically incorrect as `role=navigation` is a landmark role and should be on a `<div>` that contains the list rather than on the list itself. 

You can see an example of proper implementation here:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role
